tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -303,9 +312,9 @@ def convert_onnx_models_to_ort():
             optimization_level_str=optimization_level_str,
             optimization_style=optimization_style,
             custom_op_library=custom_op_library,
-            create_optimized_onnx_model=args.save_optimized_onnx_model,
-            allow_conversion_failures=args.allow_conversion_failures,
-            target_platform=args.target_platform,
+            create_optimized_onnx_model=save_optimized_onnx_model if save_optimized_onnx_model is not None else False,
```

Comment:
instead of having a `bool | None` type and setting a default here, should we just make the default argument value False?